roles/ftp/templates/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM stilliard/pure-ftpd:hardened
+
+ENV FTP_MAX_CLIENTS=50
+
+CMD /run.sh -c 75 -C 20 -l puredb:/etc/pure-ftpd/pureftpd.pdb -E -j -R -P $PUBLICHOST -p 30000:30099 -d -d -O w3c:/var/log/pure-ftpd/transfer.log --tls=0
```

Comment:
We probably should think about TLS. Alternatively, we can set up an IP list to only allow incoming connections from one of our NAU IP addresses.